src/applications/representatives/constants/index.js

```diff
@@ -0,0 +1,8 @@
+import * as USIP from './usip';
```

Comment:
Is this code organization idea around grouping related constants together typical or premature and therefore worth unwinding?

Future examples that aren't combining base constants like this example might look like this form from the [MDN export documentation](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export) under the `Aggregating modules` section:
```
// Aggregating modules
...
export * as name1 from "module-name";
...
```